tsl/test/expected/chunk_api-12.out

```diff
@@ -80,7 +80,8 @@ CONTEXT:  JSON data, line 1:  {"time: [1515024000000000] "device...
 -- Valid chunk, but no permissions
 SET ROLE :ROLE_DEFAULT_PERM_USER_2;
 SELECT * FROM _timescaledb_internal.create_chunk('chunkapi',' {"time": [1515024000000000, 1519024000000000], "device": [-9223372036854775808, 1073741823]}', 'ChunkSchema', 'My_chunk_Table_name');
-ERROR:  must be owner of hypertable "chunkapi"
+ERROR:  permission denied for table "chunkapi"
```

Comment:
I would expect a test where INSERT is granted to `ROLE_DEFAULT_PERM_USER_2` after this error and then try again with success.

Reply:
Fixed.